Sources/Support/Logging/OSLogStore.swift

```diff
@@ -5,7 +5,7 @@ import OSLog
 extension OSLogStore {
     
     /// Returns a sequence of log entries with timestamp after the provided date.
-    public func entries(after date: Date) throws -> AnySequence<OSLogEntry> {
+    func entries(after date: Date) throws -> AnySequence<OSLogEntry> {
```

Comment:
This is intentionally `public` and already part of the latest stable (versioned) API of the package.